tasks/src/main/java/taf/driver/DriverFactory.java

```diff
@@ -0,0 +1,28 @@
+package taf.driver;
+
+import org.openqa.selenium.WebDriver;
+
+public class DriverFactory {
```

Comment:
For increasing simplicity of using WebDriver I'd recommend you to merge BrowserFactory & DriverFactory in one class